packages/11ty/_plugins/transforms/outputs/epub/transform.js

```diff
@@ -74,6 +74,35 @@ module.exports = function(eleventyConfig, collections, content) {
       tableOfContents.setAttribute('epub:type', 'toc')
     }
 
+    /**
+     * Rewrite relative web links to work properly in epub readers
+     */
+    const pageLinks = body.querySelectorAll('a')
+    pageLinks.forEach((link) => {
+      const href = link.getAttribute('href')
+      if (!href) return
+
+      const isRelativeLink = (href) => {
+        return !href.startsWith('#') && !href.startsWith('http')
+      }
```

Comment:
@anderspollack `isRelativeLink` will evaluate to false for links like
```
<a href="#on-this-page">an anchor link on this page</a>
```
is that correct?

I would have expected this function to evaluate like so
```suggestion
      const isRelativeLink = (href) => href.startsWith('#')
```
or perhaps also checking for a fully qualified url within the publication?
```
      const isRelativeLink = (href) => 
        href.startsWith('#') || !href.match(`^https?://${baseUrl}/${pageUrl}/`)
```

Reply:
We shouldn't have to transform any pure hash links that link within the same section (like a footnote backref `<a href="#fnref5" class="footnote-backref">↩︎</a>`), since those are already working fine in EPUB output. This is filtering _only_ non-hash, non-FQDN links (like `/catalogue/1/`) for transformation, since these are now different filepaths in the EPUB package

Reply:
Renaming for clarity
```
    const links = body.querySelectorAll('a')
    links.forEach((element) => {
      const href = element.getAttribute('href')
      if (!href) return

      const isPageLink = (href) => {
        return !href.startsWith('#') && !href.startsWith('http')
      }
```
alternately
```
  body.querySelectorAll('a').forEach((element) => {
    element.getAttribute('href')
    if (!href) return

    const isPageLink = (href) => {
      return !href.startsWith('#') && !href.startsWith('http')
    }
```

Reply:
sounds good, applied this